hack/common.sh

```diff
@@ -29,12 +35,20 @@ readonly OS_SCRATCH_IMAGE_COMPILE_TARGETS=(
 )
 readonly OS_IMAGE_COMPILE_BINARIES=("${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]##*/}" "${OS_IMAGE_COMPILE_TARGETS[@]##*/}")
 
-readonly OS_CROSS_COMPILE_PLATFORMS=(
+OS_CROSS_COMPILE_PLATFORMS=(
   linux/amd64
   darwin/amd64
   windows/amd64
   linux/386
 )
+if [[ "$(os::build::host_platform)" == "linux/ppc64le" ]]; then
```

Comment:
Currently there is known issue with golang and it is failing to link bigger binaries. So I'm intentionally avoiding to consider ppc64le for cross compiling other than on ppc64le host platform.

Currently I'm running with a patched golang to build openshift or even I can use gccgo to build. Once fix is available in the golang community, I'll add `linux/ppc64` to all host platforms.